templates/fxa_users_daily_v1.sql

```diff
@@ -56,7 +56,7 @@ WITH
       'mktg - email_sent',
       'sync - repair_success',
       'sync - repair_triggered')
-    AND DATE(submission_timestamp) = @submission_date
+    AND udf_enforce_date_param(DATE(submission_timestamp), @submission_date)
```

Comment:
have you checked that this successfully reduces the bytes scanned when specified?